x-pack/test/functional/page_objects/security_page.ts

```diff
@@ -7,6 +7,7 @@
 import { adminTestUser } from '@kbn/test';
 import { FtrProviderContext } from '../ftr_provider_context';
 import { AuthenticatedUser, Role } from '../../../plugins/security/common/model';
+import { UserFormValues } from '../../../plugins/security/public/management/users/edit_user/user_form';
```

Comment:
**nit** we can use the "new" `import type` syntax here:

```suggestion
import type { UserFormValues } from '../../../plugins/security/public/management/users/edit_user/user_form';
```

Reply:
Isn't that only required for classes (i.e. when you need to tell the TypeScript compiler whether you're referring to the class or the type so that it can remove the code from the build)?

Reply:
Yeah you're right. I've come to like `import type` because it makes it clearer to a casual reader that this is in fact _just_ a TypeScript construct, and not something more. It's doubly helpful because we don't have strict naming conventions around types, interfaces, classes, etc.